### ISSUE TITLE
Got RangeError on some cases

### ISSUE THREAD

**Issue by Justineo** [ben-eb/perfectionist#2]
`test.css`:

```css
.box {
  -ms-flex-order: 1;
   -webkit-order: 1;
           order: 1;
}
```

```bash
perfectionist test.css

RangeError
    at RangeError (<anonymous>)
    at String.repeat (/usr/local/lib/node_modules/perfectionist/node_modules/string.prototype.repeat/repeat.js:26:11)
    at space (/usr/local/lib/node_modules/perfectionist/dist/space.js:17:16)
    at /usr/local/lib/node_modules/perfectionist/dist/index.js:117:62
    at Array.forEach (native)
    at /usr/local/lib/node_modules/perfectionist/dist/index.js:113:24
    at Array.forEach (native)
    at /usr/local/lib/node_modules/perfectionist/dist/index.js:107:26
    at /usr/local/lib/node_modules/perfectionist/dist/index.js:120:15
    at /usr/local/lib/node_modules/perfectionist/node_modules/postcss/lib/container.js:102:26
```

In https://github.com/ben-eb/perfectionist/blob/master/src/index.js#L83:
`vendor - thisVendor` is `-1` in this case thus `repeat` went wrong.

**Comment by ben-eb** [comment]
Thanks, I'll look into it.

**Comment by ben-eb** [comment]
Released as `1.0.1`. Thanks for reporting. :smile:

**Comment by Justineo** [comment]
:+1:
